### PR TITLE
Makes fix air z slightly more laggy but slightly safer too. Don't think this is the cause but whatever. Toggling SSs is hacky anyways. And for the record, that wasn't my idea.

### PR DESCRIPTION
**Pull request by alexkar598**
Makes fix air z slightly more laggy but slightly safer too. Don't think this is the cause but whatever. Toggling SSs is hacky anyways. And for the record, that wasn't my idea.

### CODE REVIEW

File: yogstation/code/modules/admin/verbs/fix_air.dm [hunk 1]
```diff
@@ -30,17 +30,18 @@
 	if(!check_rights(R_ADMIN))
 		return
 
+	if(Master.current_runlevel < RUNLEVEL_GAME)
+		//dipshit alert
```
Comment (Bibby0110) [inline yogstation/code/modules/admin/verbs/fix_air.dm]:
```suggestion
		//jamie alert
```

**Review by Bibby0110 — APPROVED** [review]

File: yogstation/code/modules/admin/verbs/fix_air.dm [hunk 2]
```diff
@@ -30,17 +30,18 @@
 	if(!check_rights(R_ADMIN))
 		return
 
+	if(Master.current_runlevel < RUNLEVEL_GAME)
+		//jamie alert
+		to_chat(src, "This command may not be used before the game has started!", confidential=TRUE)
+		message_admins("[src] has attempted to fix air before the game has started. Please talk some sense into them.")
```
Comment (JamieD1) [inline yogstation/code/modules/admin/verbs/fix_air.dm]:
```suggestion
		to_chat(src, "This command may not be used before the game has started!", confidential=TRUE)
		message_admins("[src] has attempted to fix air before the game has started.")
```

File: yogstation/code/modules/admin/verbs/fix_air.dm [hunk 3]
```diff
@@ -30,17 +30,18 @@
...
+		return
+
 	if(alert("Do you want to reset air on the entire z level?", "Fix Air- Z level", "No", "Yes") != "Yes")
 		return
 
 	message_admins("[key_name_admin(usr)] fixed air on zlevel [mob.z]")
 	log_game("[key_name_admin(usr)] fixed air on zlevel [mob.z]")
```
Comment (JamieD1) [inline yogstation/code/modules/admin/verbs/fix_air.dm:44]: var/atmos_enabled = SSair.can_fire
	if(atmos_enabled)
		message_admins("Disabling atmospherics to fix air on zlevel")
		SSair.can_fire = FALSE

Comment (JamieD1) [inline yogstation/code/modules/admin/verbs/fix_air.dm:44]:
```suggestion
	var/atmos_enabled = SSair.can_fire
	if(atmos_enabled)
		message_admins("Disabling atmospherics to fix air on zlevel")
		SSair.can_fire = FALSE
```

File: yogstation/code/modules/admin/verbs/fix_air.dm [hunk 4]
```diff
@@ -51,8 +52,3 @@
 			if(!istype(T, /turf/open/space) && T.is_openturf && !T.blocks_air)
 				T.air?.parse_gas_string(T.initial_gas_mix)
 				T.update_visuals()
```
Comment (JamieD1) [inline yogstation/code/modules/admin/verbs/fix_air.dm:54]:
```suggestion
				T.update_visuals()
			CHECK_TICK

	if(atmos_enabled)
		message_admins("Re-enabling atmospherics, air on zlevel fixed")
		SSair.can_fire = atmos_enabled
```

File: yogstation/code/modules/admin/verbs/fix_air.dm (hunk 3 above)
Comment (alexkar598) [reply yogstation/code/modules/admin/verbs/fix_air.dm:44]: No this is hacky and in retrospect I should have gutted this out of adam's code

File: yogstation/code/modules/admin/verbs/fix_air.dm (hunk 4 above)
Comment (alexkar598) [reply yogstation/code/modules/admin/verbs/fix_air.dm:54]: No this is hacky and in retrospect I should have gutted this out of adam's code

File: yogstation/code/modules/admin/verbs/fix_air.dm [hunk 5]
```diff
@@ -30,17 +30,18 @@
 	if(!check_rights(R_ADMIN))
 		return
 
+	if(Master.current_runlevel < RUNLEVEL_GAME)
+		//jamie alert
```
Comment (JamieD1) [inline yogstation/code/modules/admin/verbs/fix_air.dm:34]:
```suggestion
```

**Pull request merged**